libmilter.yaml

```diff
@@ -0,0 +1,40 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/libmilter/APKBUILD
+package:
+  name: libmilter
+  version: 1.0.2
+  epoch: 0
+  description: Sendmail Mail Filter API (Milter)
+  copyright:
+    - license: Sendmail
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - m4
+      - linux-headers
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 04bc76b6c886e6d111be7fd8daa32b8ce00128a288b6b52e067bc29f3854a6e6
+      uri: https://ftp.sendmail.org/sendmail.8.17.1.tar.gz
+  - runs: |
+      make -C libmilter MILTER_SOVER=${{package.version}}
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/lib
+      make -C libmilter \
+        INCOWN=root INCGRP=root INCMODE=644 \
+        LIBOWN=root LIBGRP=root LIBMODE=644 UBINMODE=755 \
+        MILTER_SOVER=${{package.version}} \
+        DESTDIR="${{targets.destdir}}" \
+        install
+  - uses: strip
+subpackages:
+  - name: libmilter-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - libmilter
```

Comment:
This shouldn't be needed anymore.  Melange figures out the dependencies of -dev packages on its own now.